### PR TITLE
ci: add support for Node 16

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/766/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/ag-grid-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/layout-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/tabs-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/766/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/766/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/766/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/766/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/766/playground/)

File: libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.spec.ts [hunk 1]
```diff
@@ -22,7 +21,7 @@ describe('SkyAgGridCellEditorLookupComponent', () => {
   ];
   let fixture: ComponentFixture<SkyAgGridCellEditorLookupComponent>;
   let nativeElement: HTMLElement;
-  let callback: EventCallback | undefined;
+  let callback: (args: Record<string, unknown>) => void | undefined;
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.spec.ts:24]: Why are these changes needed for the node version change?

Comment (Blackbaud-SteveBrush) [reply libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.spec.ts:24]: They were found during the Node 16 update process. They can technically remain, but we shouldn't be importing from `typedoc` in the libraries' source code.

Comment (Blackbaud-SteveBrush) [reply libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.spec.ts:24]: My guess is it was an auto-IDE lookup that caused the import to be added in the first place.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]
